stemcell_builder/stages/bosh_cloudstack_agent_settings/apply.sh

```diff
@@ -2,15 +2,14 @@
 
 base_dir=$(readlink -nf $(dirname $0)/../..)
 source $base_dir/lib/prelude_apply.bash
-source $base_dir/lib/prelude_agent.bash
 
 agent_settings_file=$chroot/var/vcap/bosh/agent.json
 
 cat > $agent_settings_file <<JSON
 {
   "Platform": {
     "Linux": {
-      $(get_partitioner_type_mapping)
+      "PartitionerType": "parted",
```

Comment:
Could this also be moved in to an assets file? It doesn't look like there's any more interpolation in the `<<JSON` heredoc.